### ISSUE TITLE
Special characters in the secret key cause failures

### ISSUE THREAD

**Issue by martinstreicher** [andrewrk/node-s3-cli#21]
(This is not necessarily a bug in S3 CLI, but the bug does affect it.)

If the AWS secret key contains a special character (in my case, a `[`), the s3-cli commands do not work. Instead, the commands yield the error, `Error: The request signature we calculated does not match the signature you provided. Check your key and signing method.` 

To work around the problem, issue yourself a new access key/secret key pair, where the latter does not have any special characters.

**Comment by martinstreicher** [comment]
See https://github.com/aws/aws-cli/issues/602

**Comment by guyappy** [comment]
Indeed, I changed the secret to one without special chars (just re-created one until I got one without special chars) and then it worked. 

I tried;
aws opsworks register etc.

**Comment by akhil-bp** [comment]
You shouldn't be using SMTP credentials when calling SES through an SDK, you should be using the credentials associated with an IAM account: https://docs.aws.amazon.com/ses/latest/DeveloperGuide/using-credentials.html?icmpid=docs_ses_console

Just to be sure, can you confirm you're using the correct region as well?

It might also be helpful if you can share what params you passed to sendEmail. Don't need specific values, just want to make sure I can make a request when filling out the same params.
© https://github.com/aws/aws-sdk-js/issues/1443#issuecomment-291206252
It works for me
